site/SiteResources.tsx

```diff
@@ -7,11 +7,7 @@ export const SiteResources = () => {
                 <a href="/charts">Charts and Explorers</a>
             </li>
             <li>
-                <a
-                    href="https://sdg-tracker.org"
-                    target="_blank"
-                    rel="noopener"
-                >
+                <a href="/sdgs" target="_blank" rel="noopener">
```

Comment:
Should we remove `target="_blank" rel="noopener"` ? Now that it's the same domain idk if we want it to open in a new tab.